<ui version="4.0" >
 <class>RmShaderDialogClass</class>
 <widget class="QDialog" name="RmShaderDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>364</height>
   </rect>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>RmShaderDialog</string>
  </property>
  <property name="windowIcon" >
   <iconset>../../../../../meshlab/src/meshlab/images/eye16.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Passes</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QTabWidget" name="tabWidget" >
        <property name="currentIndex" >
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab1_Uniform" >
         <attribute name="title" >
          <string>Uniform</string>
         </attribute>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QFrame" name="frameUniform" >
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth" >
             <number>1</number>
            </property>
            <property name="midLineWidth" >
             <number>0</number>
            </property>
            <layout class="QGridLayout" >
             <property name="margin" >
              <number>9</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item row="0" column="0" >
              <layout class="QGridLayout" >
               <property name="margin" >
                <number>0</number>
               </property>
               <property name="spacing" >
                <number>6</number>
               </property>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab2_Texture" >
         <attribute name="title" >
          <string>Textures</string>
         </attribute>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QFrame" name="frameTextures" >
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" >
             <property name="margin" >
              <number>9</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item row="0" column="0" >
              <layout class="QGridLayout" >
               <property name="margin" >
                <number>0</number>
               </property>
               <property name="spacing" >
                <number>6</number>
               </property>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab3_OpenGl" >
         <attribute name="title" >
          <string>OpenGL</string>
         </attribute>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QFrame" name="frameOpenGL" >
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth" >
             <number>1</number>
            </property>
            <property name="midLineWidth" >
             <number>0</number>
            </property>
            <layout class="QGridLayout" >
             <property name="margin" >
              <number>9</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item row="0" column="0" >
              <layout class="QGridLayout" >
               <property name="margin" >
                <number>0</number>
               </property>
               <property name="spacing" >
                <number>6</number>
               </property>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab4_Vertex" >
         <attribute name="title" >
          <string>Vertex</string>
         </attribute>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QTextEdit" name="textVertex" >
            <property name="lineWrapMode" >
             <enum>QTextEdit::NoWrap</enum>
            </property>
            <property name="readOnly" >
             <bool>true</bool>
            </property>
            <property name="acceptRichText" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab5_Fragment" >
         <attribute name="title" >
          <string>Fragment</string>
         </attribute>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QTextEdit" name="textFragment" >
            <property name="lineWrapMode" >
             <enum>QTextEdit::NoWrap</enum>
            </property>
            <property name="readOnly" >
             <bool>true</bool>
            </property>
            <property name="acceptRichText" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QComboBox" name="cmbPassSelection" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="lblEffectSelection" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>4</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Effect Selection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbEffectSelection" />
     </item>
     <item>
      <widget class="QPushButton" name="btnOk" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
